docs/tutorials/routing.rst

```diff
@@ -387,31 +387,22 @@ We can now start BIRD inside the bgp namespace:
     as_ns bgp bird -P /run/bird-bgp.pid
 
 We'll configure Faucet to talk to BIRD by adding BGP configuration to
-``/etc/faucet/faucet.yaml``. Change the servers VLAN to look like the
-configuration below, leaving all other VLANs alone, and add a Faucet
-router.
+``/etc/faucet/faucet.yaml``. Replace the routers section with the below.
 
 .. code-block:: yaml
     :caption: /etc/faucet/faucet.yaml
 
     routers:
         bird:
-            vlans: servers
+            vlans: [hosts, servers]
```

Comment:
I wonder if it might be a little clearer to have two routers, one for the BGP peering and one to do inter-vlan routing between the two subnets, a sample config could look like:

```
routers:
    router-hosts-servers:
        vlans: [servers, hosts]
    bird:
        bgp:
            vlan: servers
            as: 65000                           # Faucet's AS number
            port: 9179                          # BGP port for Faucet to listen on.
            routerid: '10.0.1.3'                # Faucet's Unique ID.
            server_addresses: ['10.0.1.3']      # Faucet's listen IP for BGP
            neighbor_addresses: ['10.0.1.2']    # Neighbouring IP addresses (IPv4/IPv6)
            neighbor_as: 65001                  # Neighbour's AS number
```